e2e_playwright/st_audio_input_test.py

```diff
@@ -238,6 +237,8 @@ def test_audio_input_works_in_forms(app: Page):
     submit_button.scroll_into_view_if_needed()
     expect(submit_button).to_be_enabled()
 
+    app.wait_for_timeout(1500)
+
     # Verify the form state has not changed yet
     expect(app.get_by_text("Audio Input in Form: None")).to_be_visible()
```

Comment:
If the issue is mostly about the the timing, it would be better to just use a higher timeout in `to_be_visible`. I think the default time the `expect(app.get_by_text("Audio Input in Form: None")).to_be_visible()` checks for the appearance is 5 seconds. Maybe use `to_be_visible(timeout=10000)` instead.

Reply:
I don't think that's the issue, I think instead it's clicking the submit button too quickly. I could move the sleep below this line to make it more clear, though.

Reply:
I see 👍  I think it would be good to add a brief comment in the code to explain why the timeout is needed.